solutions/요세푸스.js

```diff
@@ -0,0 +1,32 @@
+const INPUT_PATH = "../inputs/요세푸스.txt";
+
+const fs = require("fs");
+const filePath = process.platform === "linux" ? "/dev/stdin" : INPUT_PATH;
+const input = fs.readFileSync(filePath).toString().trim().split("\n");
+
+function solution(input) {
+  const [total, interval] = input[0].split(" ").map(Number);
+  const queue = new Array(total).fill(0).map((v, i) => i + 1);
+  const answer = [];
+  let count = 1;
+
+  while (queue.length) {
+    if (count === interval) {
+      const out = queue.shift();
+      answer.push(out);
+      count = 1;
+
+      continue;
+    }
+
+    const num = queue.shift();
+    queue.push(num);
```

Comment:
```suggestion
    const num = queue.shift();
    
    queue.push(num);
```
저는 선언부와 비즈니스 로직은 한줄 개행하는 스타일인데
수진님은 붙혀서 작성하시는군요!

Reply:
이것은 제 실수인 것 같네요! 확인하였습니다. 감사합니다 :)